src/assets.zig

```diff
@@ -427,14 +545,40 @@ pub fn loadWorldAssets(assetFolder: []const u8, blockPalette: *Palette, biomePal
 			main.utils.file_monitor.listenToPath(path, main.blocks.meshes.reloadTextures, 0);
 		}
 	}
+
+	std.log.info(
+		"Finished registering assets with {} blocks ({} migrations), {} items {} tools. {} biomes, {} recipes",
+		.{
+			blocks.count(),
+			blocksMigrations.count(),
+			items.count(),
+			tools.count(),
+			biomes.count(),
+			recipes.count()
+		},
+	);
 }
 
 pub fn unloadAssets() void { // MARK: unloadAssets()
+	std.log.info(
+		"Unloading assets: {} blocks ({} migrations), {} items {} tools. {} biomes, {} recipes",
+		.{
+			commonBlocks.count(),
+			commonBlocksMigrations.count(),
+			commonItems.count(),
+			commonTools.count(),
+			commonBiomes.count(),
+			commonRecipes.count()
+		},
+	);
```

Comment:
I think this log statement is unnecessary and also could be misleading (since the amounts shown, are the entries that are not unloaded)